drivers/bridge/setup_ip_tables.go

```diff
@@ -149,7 +149,7 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 			iptables.Raw(append([]string{"-D", chain}, dropArgs...)...)
 
 			if !iptables.Exists(table, chain, acceptArgs...) {
-				if output, err := iptables.Raw(append([]string{"-A", chain}, acceptArgs...)...); err != nil {
```

Comment:
Right -- I see. I misread your message.
In that case, the only change I will be making then is the one that aboch suggested: https://github.com/docker/libnetwork/pull/409#discussion_r35438525

Sound good?